logstash/docs.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/rakelib/plugins_docs_dependencies.rake b/rakelib/plugins_docs_dependencies.rake
+index 45e7b0884..4c2d711be 100644
+--- a/rakelib/plugins_docs_dependencies.rake
++++ b/rakelib/plugins_docs_dependencies.rake
+@@ -128,9 +128,9 @@ task :generate_plugins_version do
+   require "bundler"
+   require "bundler/dsl"
+   require "json"
+-  Bundler.setup(:default)
++  #Bundler.setup(:default)
```

Comment:
Should we just remove these lines in the patch instead of patching to comment-out?

Incidentally, I would also prefer a more descriptive name for this patch; maybe:

~~~
logstash/remove-setup-and-bootstrap-from-docs-rakelib.patch
~~~

Alternatively, a less fragile approach could be to edit the source to skip bundler setup and environment bootstrap conditionally based on the presence of an environment variable.

Reply:
I agree the patch should be avoided, I just took the existing process and moved it into a bash script. We can iterate on patching elastic/logstash branches in a second iteration, IMO